client/state/importer-nux/actions.js

```diff
@@ -57,23 +58,38 @@ export const fetchIsSiteImportable = site_url => dispatch => {
 	return wpcom
 		.undocumented()
 		.isSiteImportable( site_url )
-		.then( ( { engine, favicon, site_title: siteTitle, site_url: siteUrl } ) =>
-			dispatch(
-				setImportOriginSiteDetails( {
-					engine,
-					favicon,
-					siteTitle,
-					siteUrl,
-				} )
-			)
+		.then(
+			( {
+				site_engine: siteEngine,
+				site_favicon: siteFavicon,
+				site_title: siteTitle,
+				importer_types: importerTypes,
+			} ) => {
+				if ( siteEngine && importerTypes.length ) {
+					return dispatch(
+						setImportOriginSiteDetails( {
+							siteEngine,
+							siteFavicon,
+							siteTitle,
+							importerTypes,
+						} )
+					);
+				}
+				// If importerTypes is empty, the remote site isn't supported
+				// @todo: improve handling
+				return dispatch( {
+					type: IMPORT_IS_SITE_IMPORTABLE_ERROR,
+					error: { code: SITE_IMPORTER_ERR_BAD_REMOTE },
```

Comment:
Since we're not depending on site importer to determine importabability, I think these constants like, `SITE_IMPORTER_ERR_BAD_REMOTE` should to be renamed.